File: src/targets/php/guzzle/fixtures/text-plain.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
  'body' => 'Hello World',
  'headers' => [
    'content-type' => 'text/plain',
  ],
]);

echo $response->getBody();

File: src/targets/dart/http/fixtures/text-plain.dart
import 'package:http/http.dart' as http;

void main() async {

  final headers = {
    'content-type': 'text/plain',
  };

  final response = await http.post(
    Uri.parse('http://mockbin.com/har'),
    headers: headers,
    body: "Hello World",
  );

  print(response.statusCode);
  print(response.body);

}